continuous_integration/travis/run_test.sh

```diff
@@ -26,6 +26,9 @@ if [ "$USE_CONDA" = true ] ; then
 else
     cd $FULL_SPYDER_CLONE
     pip install dist/spyder-*.whl
+
+    # Install testing packages
+    pip install -e .[test]
```

Comment:
Is this line necessary? I mean, what I want is to install our wheel and see if it pulls our test dependencies. So something like

     pip install dist/spyder-*.whl[test]

should work (but I'm not sure :-)